python/tvm/relay/op/contrib/tensorrt.py

```diff
@@ -709,7 +709,9 @@ def strided_slice_checker(
     if any([x is not None and x <= 0 for x in attrs.strides]):
         logger.info(f"{op_name}: stride must be positive")
         return False
-    for i in range(0, len(args[0].checked_type.shape)):
+    if len(attrs.axes) > len(args[0].checked_type.shape):
+        logger.info(f"{op_name}: number of indexed axes is higher than input tensor rank")
+    for i in range(0, len(attrs.axes)):
         begin = int(attrs.begin[i])
```

Comment:
it should be `attrs.begin[attrs.axes[i]]`

Reply:
wouldn't this lead to an error if, e.g., attrs={begin: [0], end: [1], axes: [2]}?